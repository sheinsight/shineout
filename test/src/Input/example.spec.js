import exampleTest from '../../example'

describe('Input[snapshot]', () => {
  exampleTest('Input')
})
