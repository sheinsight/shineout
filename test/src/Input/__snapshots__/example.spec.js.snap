// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test: Input[site/pages/components/Input/example-1-base.js] 1`] = `
<label>
  <input />
</label>
`;

exports[`Snapshot test: Input[site/pages/components/Input/example-1-clearable.js] 1`] = `
<label>
  <input />
</label>
`;

exports[`Snapshot test: Input[site/pages/components/Input/example-1-underline.js] 1`] = `
<label>
  <input />
</label>
`;

exports[`Snapshot test: Input[site/pages/components/Input/example-2-size.js] 1`] = `
<div>
  <label>
    <input />
  </label>
  <label>
    <input />
  </label>
  <label>
    <input />
  </label>
</div>
`;

exports[`Snapshot test: Input[site/pages/components/Input/example-3-number.js] 1`] = `
<div>
  <label>
    <input />
  </label>
  <label>
    <input />
  </label>
  <label>
    <input />
  </label>
  <label>
    <input />
  </label>
  <label>
    <input />
  </label>
</div>
`;

exports[`Snapshot test: Input[site/pages/components/Input/example-4-number.js] 1`] = `
<label>
  <input />
  <a>
    <svg>
      <path />
      <path />
    </svg>
  </a>
  <a>
    <svg>
      <path />
      <path />
    </svg>
  </a>
</label>
`;

exports[`Snapshot test: Input[site/pages/components/Input/example-5-group.js] 1`] = `
<div>
  <div>
    <i />
    <label>
      <input />
    </label>
    <span>
      -
    </span>
    <label>
      <input />
    </label>
  </div>
  <div>
    <label>
      <input />
    </label>
    <button>
      <i />
    </button>
  </div>
  <div>
    <label>
      <input />
    </label>
    <label>
      <input />
    </label>
  </div>
  <div>
    <label>
      <input />
    </label>
    <button>
      <span>
        Search
      </span>
    </button>
  </div>
  <div>
    <b>
      <i />
    </b>
    <label>
      <input />
    </label>
    <b>
      .com
    </b>
  </div>
</div>
`;

exports[`Snapshot test: Input[site/pages/components/Input/example-6-tip.js] 1`] = `
<div>
  <label>
    <input />
  </label>
  <div>
    <i />
    <label>
      <input />
    </label>
  </div>
  <div>
    <i />
    <label>
      <input />
    </label>
  </div>
</div>
`;

exports[`Snapshot test: Input[site/pages/components/Input/example-7-validate.js] 1`] = `
<label>
  <input />
</label>
`;

exports[`Snapshot test: Input[site/pages/components/Input/example-8-disabled.js] 1`] = `
<div>
  <div>
    <label>
      <input />
    </label>
    <span>
      -
    </span>
    <label>
      <input />
    </label>
  </div>
  <label>
    <input />
  </label>
</div>
`;

exports[`Snapshot test: Input[site/pages/components/Input/example-9-password.js] 1`] = `
<label>
  <input />
</label>
`;
